test_api.py

```diff
@@ -19,10 +19,10 @@
 #credentials 
 uid     = os.getenv('userid')
 pwd     = os.getenv('password')
-factor2 = os.getenv('totp')
-vc      = otp
+vc      = os.getenv('vendor_code')
 app_key = os.getenv('api_key')
 imei    = <imei>
+factor2 = otp
```

Comment:
Instead of pushing commits with issues and separate commits to fix them,
please consider rebasing your changes into logical commits.